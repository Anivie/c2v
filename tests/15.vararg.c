struct JNINativeInterface_ {
    int (__stdcall *NewObject)
    (char* name, ...);
};