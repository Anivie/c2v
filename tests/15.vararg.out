[translated]
module main

struct JNINativeInterface_ { 
	NewObject fn (&i8, ... voidptr) int
}
